src/DynamoCoreWpf/UI/Themes/Modern/Ports.xaml

```diff
@@ -67,7 +67,7 @@
                 <Grid.ToolTip>
                     <dynui:DynamoToolTip AttachmentSide="{Binding Path=PortType, Converter={StaticResource PortToAttachmentConverter}}" Style="{DynamicResource ResourceKey=SLightToolTip}">
                         <Grid>
-                            <TextBlock Text="{Binding Path=ToolTipContent}"></TextBlock>
+                            <TextBlock MaxWidth="320" TextWrapping="Wrap" Text="{Binding Path=ToolTipContent}"></TextBlock>
```

Comment:
Can you elaborate here how did you define `320`? If possible can you move the number to be a const number property?

Reply:
I found that the node Rectangle ToolTip was set width as 320. (https://github.com/DynamoDS/Dynamo/blob/8f3d5305906b3effe762f428c693a10ea4ce65be/src/DynamoCoreWpf/Views/Core/NodeView.xaml#L240)
So I just keep the same as it.